docs/man/docker-cp.1.md

```diff
@@ -2,18 +2,56 @@
 % Docker Community
 % JUNE 2014
 # NAME
-docker-cp - Copy files/folders from the PATH to the HOSTPATH, or STDOUT
+docker-cp - Copy files or folders from a container's PATH to a HOSTDIR
+or to STDOUT.
 
 # SYNOPSIS
 **docker cp**
 [**--help**]
-CONTAINER:PATH HOSTPATH|-
+CONTAINER:PATH HOSTDIR|-
 
 # DESCRIPTION
-Copy files/folders from a container's filesystem to the
-path. Use '-' to write the data as a tar file to STDOUT.
-Paths are relative to the root of the filesystem. Files
-can be copied from a running or stopped container.
+
+Copy files or folders from a `CONTAINER:PATH` to the `HOSTDIR` or to STDOUT. 
+The `CONTAINER:PATH` is relative to the root of the container's filesystem. You
+can copy from a running or stopped container. 
+
+The `PATH` can be a file or directory. The `docker cp` command assumes all paths
```

Comment:
perhaps use "PATHs" instead of "paths" at the end, otherwise people may think it applies to the HOSTDIR too.